### PR TITLE
feat(doc): add category param in get fee

### PR DESCRIPTION
**Pull request by Mbomain**

### CODE REVIEW

File: src/test/java/school/hei/haapi/service/FeeServiceTest.java [hunk 1]
```diff
@@ -249,11 +249,11 @@ void fees_by_status_with_exceeded_page() {
...
 
     List<Fee> actualPaidPage1 =
-        subject.getFees(page1, pageSize, null, null, PAID, null, null, false, null);
+        subject.getFees(page1, pageSize, null, null, PAID, null, null, null, false, null);
     List<Fee> actualLatePage1 =
-        subject.getFees(page1, pageSize, null, null, LATE, null, null, false, null);
+        subject.getFees(page1, pageSize, null, null, LATE, null, null, null, false, null);
     List<Fee> actualLatePage2 =
-        subject.getFees(page2, pageSize, null, null, LATE, null, null, false, null);
+        subject.getFees(page2, pageSize, null, null, LATE, null, null, null, false, null);
```
Comment (YumeT023) [inline src/test/java/school/hei/haapi/service/FeeServiceTest.java:256]: missing test with non-null fee category

**Review by YumeT023 — CHANGES_REQUESTED** [review]

**Review by YumeT023 — APPROVED** [review]
